bindings/nodejs/benchmark/deno.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Operator } from 'npm:opendal'
```

Comment:
We must load from `npm`? Maybe it's more useful to test againest local build?

Reply:
local import is not supported yet. 
https://github.com/napi-rs/napi-rs/issues/900#issuecomment-1271821940

So i think we could keep it simple for now. Once napi-rs has a batter support for deno ffi, we could opt-in later.

Reply:
Understood. Could you please leave a comment with a link to https://github.com/napi-rs/napi-rs/issues/900 here? This will allow us to migrate once napi is released.